src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -1424,7 +1471,23 @@ private static Set<String> validateTableProperties(Set<String> properties, Strin
         Set<String> diff = Sets.difference(lowerCaseProperties, TableAttributes.allKeywords());
 
         if (!diff.isEmpty())
-            throw new IllegalArgumentException(format("Invalid value for %s: '%s' do not parse as valid table properties",
+            throw new IllegalArgumentException(format("Invalid values for %s: '%s' do not parse as valid table properties",
```

Comment:
@aparnanaik0522  you have changed it here but you have not reflected this change into table tests